lib/conrad/collector.rb

```diff
@@ -37,6 +37,10 @@ class << self
       # instead be sent one-by-one. Defaults to false.
       attr_accessor :default_emit_as_batch
 
+      # Allows assigning a default logger to use for logging out of the gem.
+      # Should respond to #debug, #info, #warn, #error, and #fatal.
+      attr_accessor :default_logger
```

Comment:
Do we need 2 loggers? Should it be collector or gem level?

Reply:
You're right, this should probably be moved to a more gem level configuration. The only possibility I can think of where that isn't better is someone using both the Recorder and the Collector and only want log events from one and not both.

Reply:
Is that ever a thing people would want? Not to dismiss the idea, but i haven't seen a single gem with multiple loggers. Is there precedent or a use case you have in mind?

Reply:
No specific use case, just throwing out the possibility. Again: i'm in favor of moving this to be a gem-level configuration, just thinking out loud a bit.